packages/messaging-api-telegram/src/__tests__/TelegramClient.spec.js

```diff
@@ -224,14 +227,19 @@ describe('send api', () => {
               height: 333,
             },
           ],
+          caption: 'gooooooodPhoto',
         },
       };
 
       mock.onPost('/sendPhoto').reply(200, reply);
```

Comment:
We should add second argument to make sure it call post with specified body:
https://github.com/ctimmerm/axios-mock-adapter/blob/master/src/index.js#L55